app/controllers/post.js

```diff
@@ -26,6 +26,15 @@ module.exports = {
       }
     })
   },
+  delete: function (req, res, next) {
+    userModel.findByIdAndRemove(req.params.id, function (err) {
+      if (err) {
```

Comment:
@tanujvyas10 if there is any error just throw it

Reply:
Required changes have been made.Please review

Reply:
what are you trying to delete here?

Reply:
you are consuming User model here why?
Also, task will be deleted with the help of task_id